src/Controls/tests/Core.UnitTests/GridTests.cs

```diff
@@ -901,9 +901,7 @@ public void AddDimensionTheory(string operations)
 				var id = 0;
 				foreach (var view in _grid.Children.Cast<Label>().OrderBy(o => o.Text))
 				{
-					var expected = $"{id++}: " +
-						$"{Grid.GetColumn(view)}x{Grid.GetRow(view)} " +
-						$"{Grid.GetColumnSpan(view)}x{Grid.GetRowSpan(view)}";
+					var expected = $"{id++}: {Grid.GetColumn(view)}x{Grid.GetRow(view)} {Grid.GetColumnSpan(view)}x{Grid.GetRowSpan(view)}";
```

Comment:
Not strictly related to the failing tests, but this allows us to get a green build until we get the fixes for https://github.com/dotnet/roslyn/pull/55494